Tests/Fixtures/ClassParser/ComplexClass.php

```diff
@@ -18,14 +18,14 @@
  */
 require_once(\TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath('extension_builder') . 'Tests/Fixtures/ClassParser/BasicClass.php');
 
+const testConstant = '123';
+const testConstant2 = 0.56;
```

Comment:
what are these constants doing outside of the class?

Reply:
They are used in the fixture class as value - just some "noise" for the class parser test

Reply:
Since there are so many possible things to test in the parser I have to admit this is just a random selection of what could be tested...

Reply:
I don't see any test cases for those constants since before they were inside the class, now they are outside.